assets/src/edit-story/components/library/libraryPanes.js

```diff
@@ -20,16 +20,23 @@
 import useLibrary from './useLibrary';
 import { getPane } from './panes';
 import { getTabId } from './panes/shared';
+import { Media3pApiProvider } from './panes/media/media3p/api';
 
 function LibraryPanes() {
   const { tab, tabs } = useLibrary((state) => ({
     tab: state.state.tab,
     tabs: state.data.tabs,
   }));
   const panes = tabs.map(getPane);
-  return panes.map(({ id, Pane }) => (
-    <Pane key={id} isActive={id === tab} aria-labelledby={getTabId(id)} />
-  ));
+
+  // TODO(diegovar): Potentially move Media3pApiProvider down.
+  return (
+    <Media3pApiProvider>
```

Comment:
Why not leave this with the other provider objects?
https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/app/index.js

Reply:
This will need to be added above <MediaProvider>, so that the listMedia function can be accessed from MediaProvider's fetch side effect.